test/Dapr.Client.Test/PublishEventApiTest.cs

```diff
@@ -294,6 +295,37 @@ public async Task PublishEventAsync_CanPublishWithRawData()
         envelope.Metadata.Count.ShouldBe(0);
     }
 
+    [Fact]
+    public async Task PublishBulkByteEventAsync_PublishesEventsAndReturnsResponse()
+    {
+        // Arrange
+        var client = new DaprClientBuilder().Build();
```

Comment:
Unit tests aren't run with a running instance of Dapr, so this isn't going to work correctly. I would advise copying the example above using the `TestClient` it produces so you can monitor the "captured" gRPC request and validate against that payload.